resources/js/Pages/Dashboard/CourseSemester/Edit.vue

```diff
@@ -34,40 +76,37 @@ function updateCourse() {
     <div class="flex flex-col gap-8">
       <ManagementHeader>
         <template #header>
-          Zarządzanie kursami w semestrze
+          Podgląd i zarządzanie kursem w semestrze
         </template>
         <template #statistics>
           <ManagementHeaderItem>
-            Formularz edycji istniejącego kursu w semestrze
+            Formularz podglądu kursu w semestrze
           </ManagementHeaderItem>
         </template>
       </ManagementHeader>
-      <form class="grid grid-cols-2" @submit.prevent="updateCourse">
+
+      <form class="grid grid-cols-2" @submit.prevent="updateCourse()">
         <Section>
           <div class="flex flex-col justify-between gap-4">
             <FormGroup>
               <FormLabel for="id">
                 Id
               </FormLabel>
-              <TextInput class="opacity-75" placeholder="autogenerowany ulid" autocomplete="off" disabled />
+              <TextInput class="opacity-75" :placeholder="course.id" autocomplete="off" disabled />
```

Comment:
Maybe on another edit views we should remove text "autogenerowany ulid" from placeholder and add resource id like herE?

Reply:
Changed.